### PR TITLE
fix: Add defaults to console command generate

### PR DESCRIPTION
**Pull request by bhaskarmelkani**
Add defaults to console command `generate`.

Related to [#Issue](https://github.com/hexojs/hexo/issues/2851)

### CODE REVIEW

**Comment by bhaskarmelkani** [comment]
Labelled bug as tests were not in sync with code.

File: test/scripts/console/generate.js [hunk 1]
```diff
@@ -20,7 +20,6 @@ describe('generate', () => {
   afterEach(() => fs.rmdir(hexo.base_dir));
 
   function testGenerate(options) {
-    options = options || {};
```
Comment (JLHwung) [inline test/scripts/console/generate.js]: `%s/generate(\{\})/generate()/g`.

Comment (bhaskarmelkani) [reply test/scripts/console/generate.js]: Oh yeah, makes sense. (y)

**Comment by JLHwung** [comment]
Thank you so much for you contribution. Hexo can't be rock without contributors like you.

**Pull request merged**